src/main/groovy/org/mockito/release/internal/gradle/ContinuousDeliveryPlugin.java

```diff
@@ -46,6 +46,19 @@ public void apply(final Project project) {
         project.getPlugins().apply(VersioningPlugin.class);
         project.getPlugins().apply(GitPlugin.class);
 
+        project.allprojects(new Action<Project>() {
+            @Override
+            public void execute(final Project project) {
+                project.getPlugins().withType(JavaLibraryPlugin.class, new Action<JavaLibraryPlugin>() {
+                    @Override
+                    public void execute(JavaLibraryPlugin plugin) {
+                        project.getTasks().getByName("publishToMavenLocal")
```

Comment:
It is safer to find the exact task that generates pom file and leverages customizePom method. Otherwise fetchAllProjectContributorsFromGitHub might get executed too late, after the poms are generated but before publishToMavenLocal runs.